doc/rtd/topics/tutorials/qemu.rst

```diff
@@ -0,0 +1,282 @@
+.. _tutorial_qemu:
+
+Qemu Tutorial
+*************
+
+.. toctree::
+   :titlesonly:
+   :hidden:
+
+   qemu-debugging.rst
+
+
+Tutorial overview
+=================
```

Comment:
```suggestion
```

Reply:
This looks a bit odd when rendered in RTD because it's so close to the page title. I'd recommend just taking it out, since it's not really needed.